src/CoreWCF.RabbitMQ/tests/ServiceHelper.cs

```diff
@@ -19,7 +19,7 @@ public static IWebHostBuilder CreateWebHostBuilder<TStartup>(ITestOutputHelper o
 #if DEBUG
                 .ConfigureLogging((ILoggingBuilder logging) =>
                 {
-                    logging.AddProvider(new XunitLoggerProvider(outputHelper));
+                    logging.AddProvider(new XunitLoggerProvider(outputHelper, nameof(CreateWebHostBuilder)));
                     logging.AddFilter("Default", LogLevel.Debug);
```

Comment:
This is supposed to be the name of the test. Have a look how it's used in other repos. We add a parameter to the CreateDefaultBuilder method which is attributed to be filled in by the compiler with the calling method name.